test/number-test.js

```diff
@@ -0,0 +1,128 @@
+var nock = require("nock");
+var CatapultClient = require("../index");
+
+var baseUrl = "https://api.catapult.inetwork.com";
+
+describe("PhoneNumber API", function () {
+
+	describe("global methods", function () {
+		var client;
+
+		var userId = "fakeUserId";
+		var apiToken = "fakeApiToken";
+		var apiSecret = "fakeapiSecret";
+
+		var phoneNumberList = [
+      {
+	"id"             : "{numberId1}",
+	"application"    : "https://catapult.inetwork.com/v1/users/users/u-ly123/applications/a-j321",
+	"number"         :"{number1}",
+	"nationalNumber" :"{national_number1}",
+	"name"           : "home phone",
+	"createdTime"    : "2013-02-13T17:46:08.374Z",
+	"state"          : "NC",
+	"price"          : "0.60",
+	"numberState"    : "enabled"
+      },
+      {
+	"id"             : "{numberId2}",
+	"application"    : "https://catapult.inetwork.com/v1/users/users/u-ly123/applications/a-j123",
+	"number"         :"{number2}",
+	"nationalNumber" :"{national_number2}",
+	"name"           : "work phone",
+	"createdTime"    : "2013-02-13T18:32:05.223Z",
+	"state"          : "NC",
+	"price"          : "0.60",
+	"numberState"    : "enabled"
+      }
+    ];
+
+		before(function () {
+			client = new CatapultClient({
+				userId    : userId,
+				apiToken  : apiToken,
+				apiSecret : apiSecret
+			});
+			nock.disableNetConnect();
+
+			nock(baseUrl)
+				.persist()
+				.post("/v1/users/" + userId + "/phoneNumbers")
+				.reply(201, {},
+          {
+	"Location" : "/v1/users/" + userId + "/phoneNumbers/fakeNumberId"
+          }
+        )
+				.get("/v1/users/" + userId + "/phoneNumbers")
+				.reply(200, phoneNumberList);
+		});
+
+		after(function () {
+			nock.cleanAll();
+			nock.enableNetConnect();
+		});
+
+		it("should get a list of phone numbers, promise style", function (done) {
+			client.PhoneNumber.list()
+			.then(function (numberResponse) {
```

Comment:
prefer to use deep equal here 

or break this down into individual `it` statements. Ideally, only one assertion per `it` block if possible. Helps trace down future errors.